src/select/parts/virtual-list.tsx

```diff
@@ -68,12 +77,19 @@ const VirtualListOpen = forwardRef(
       useInteractiveGroups,
       screenReaderContent,
       ariaSetsize: filteredOptions.length,
+      withScrollbar: hasScrollbar,
+      firstOptionSticky,
     });
 
     return (
       <OptionsList {...menuProps} ref={menuRef}>
-        <div aria-hidden="true" key="total-size" className={styles['layout-strut']} style={{ height: totalSize }} />
         {finalOptions}
+        <div
```

Comment:
Moved the strut after the options so that sticky options stick "naturally" with `position: sticky`, otherwise they would get pushed to the bottom.

This is similar to the fact that the bottom option `listBottom` is render after the strut. So now the strut is in between.